code/game/objects/items/storage/boxes.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+const {Component, Atom} = require('bluespess');
+
+class Box extends Component {
+	constructor(atom, template) {
+		super(atom, template);
+		if(this.has_illustration) {
```

Comment:
I don't think you need a seperate  `has_illustration`, just do `if(this.illustration)`.